shared/message/support/Gazebo/GazeboBallLocation.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package message.support.Gazebo;
+
+message GazeboBallLocation {
+
+    double x = 1;
```

Comment:
We have vec3s for this. Also, this should be a vector of the form rABc so we know what space it's measured in etc

Reply:
you think we know what space this is in?